testutil/util.go

```diff
@@ -78,3 +109,37 @@ func Cleanup(t *testing.T, options *terraform.Options) {
 	terraform.Destroy(t, options)
 	Clean(options.TerraformDir)
 }
+
+func ListEnvVar(name string) []string {
+	s := os.Getenv(name)
+	return strings.Split(s, ",")
+}
+
+func EnvVar(name string) string {
+	return os.Getenv(name)
+}
+
+// Lifted from https://github.com/gruntwork-io/terratest/blob/master/modules/random/random.go and modified
+// to do lowercase only.
+// TODO fork terratest and write a method for random strings that takes a list of acceptable characters.
+const base62chars = "abcdefghijklmnopqrstuvwxyz"
+const uniqueIDLength = 6 // Should be good for 62^6 = 56+ billion combinations
+
+// UniqueId returns a unique (ish) id we can attach to resources and tfstate files so they don't conflict with each other
+// Uses base 62 to generate a 6 character string that's unlikely to collide with the handful of tests we run in
+// parallel. Based on code here: http://stackoverflow.com/a/9543797/483528
+func UniqueId() string {
+	var out bytes.Buffer
+
+	generator := newRand()
+	for i := 0; i < uniqueIDLength; i++ {
+		out.WriteByte(base62chars[generator.Intn(len(base62chars))])
+	}
+
+	return out.String()
+}
+
+// newRand creates a new random number generator, seeding it with the current system time.
+func newRand() *rand.Rand {
```

Comment:
idk if we'd ever want to print the seed to repro

Reply:
Yeah I can add that to the improvements we'd want to make to terratest, ok?

Reply:
:+1: